npm-shrinkwrap.json

```diff
@@ -2600,9 +2600,9 @@
       }
     },
     "appium-xcuitest-driver": {
-      "version": "2.113.2",
-      "resolved": "https://registry.npmjs.org/appium-xcuitest-driver/-/appium-xcuitest-driver-2.113.2.tgz",
-      "integrity": "sha512-dhid34cx99AG61O69gbIgvW51RUKtd9uQkKziQe0/XGApAozjwlxgMJo/pjshpuz2rXCMz3ilRcrLnscT36qmQ==",
+      "version": "2.114.0",
```

Comment:
is it greater to bump https://github.com/appium/appium-xcuitest-driver/commits/master again and set here as 2.114.3? (They have fixes for WDA building/signing)